guides/inference-scheduling/ms-inference-scheduling/values_cpu.yaml

```diff
@@ -0,0 +1,123 @@
+# Simplified Intel XPU configuration using imageDefault mode
+# This configuration lets the chart handle most vLLM parameters automatically
+
+modelArtifacts:
+  uri: "hf://meta-llama/Llama-3.2-3B-Instruct"
+  size: 100Gi
+  authSecretName: "llm-d-hf-token"
+  name: "meta-llama/Llama-3.2-3B-Instruct"
+
+
+accelerator:
+  type: cpu
+
+# Routing configuration - required for inference-scheduling
+routing:
+  servicePort: 8000
+  proxy:
+    image: ghcr.io/llm-d/llm-d-routing-sidecar:v0.3.0
```

Comment:
The sidecar is only needed in P/D

Reply:
didn't realize we are using the sidecar in this user guide, which is unnecessary. I am OK for now and we can clean it up as a follow up./